Sources/MisticaCommon/Components/Tag/TagStyle.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  TagStyle.swift
+//
+//  Made with ❤️ by Novum
+//
+//  Copyright © Telefonica. All rights reserved.
+//
+
+import Foundation
+
+import UIKit
+
+@frozen
+public enum TagStyle: String, CaseIterable {
+    case promo,
+         active,
+         inactive,
+         success,
+         warning,
+         error
+}
+
+public extension TagStyle {
+    func backgroundColor(_ isInverse: Bool) -> UIColor {
+        if isInverse {
+            return inverseBackgroundColor
+        } else {
+            return defaultBackgroundColor
+        }
```

Comment:
I would prefer this but minor...
```suggestion
     isInverse ? inverseBackgroundColor : defaultBackgroundColor
```